src/lib/database/seed-from-live-api.ts

```diff
@@ -0,0 +1,40 @@
+import type { Routes } from "@seamapi/types/devicedb"
+import type { AxiosInstance } from "axios"
+import type { Simplify } from "type-fest"
+import type { TypedAxios } from "typed-axios-instance"
+
+import type { Database } from "./schema.ts"
+
+interface PopulateFromLiveApiOptions {
+  db: Database
+  live_client: AxiosInstance
+  device_category: Routes["/v1/device_models/list"]["queryParams"]["main_category"]
+}
+
+export const seedFromLiveApi = async ({
+  db,
+  live_client,
+  device_category,
+}: PopulateFromLiveApiOptions) => {
```

Comment:
Since it's not going on the fake (or either way actually), it should match the name / signature of `seedDatabase`:

```suggestion
export const seedDatabaseFromLiveApi = async (db, client, {
  device_category = 'lock', // lets make this 3rd argument optional
}: PopulateFromLiveApiOptions) => {
```

Reply:
not sure it matters too much but 🤷

Reply:
3bc514c